ui-tests/pageobjects/page.ts

```diff
@@ -11,6 +11,10 @@ export default class Page {
     return $(".govuk-heading-l");
   }
 
+  public get pageParagraphText(): Promise<WebdriverIO.Element> {
+    return $("p.govuk-body");
+  }
```

Comment:
be wary, there's likely to be ,multiple <p>s on a page

Reply:
using this function on homepage and currently we have only <p> on that page